Updating README to remove the optionals

CheezItMan opened the pull request:
Adjusting Optionals and feedback template and I added a link to the feedback template in the README.  

Feel free to adjust from here, I just had 5 minutes between 1-on-1s.

CheezItMan committed:
Updating README to remove the optionals

Also using table shorthand

CheezItMan committed:
Update README.md

CheezItMan committed:
Updating Feedback Table

CheezItMan committed:
Additional notes

droberts-sea commented on feedback.md:
```diff
@@ -1,16 +1,17 @@
-|  Planet Class Created |  |
-|  Created a collection of Planet objects as an instance variable in SolarSystem. |   |   
+|  Feature 	|  Feedback
+|---	|---	
```
You can get rid of the leading pipe-space-space here as well

droberts-sea reviewed (approved):
Looks good. I think Dee has some outstanding changes on this project, so you might want to double check with her.

CheezItMan committed:
Update PULL_REQUEST_TEMPLATE

CheezItMan committed:
Update feedback.md

droberts-sea commented on .github/PULL_REQUEST_TEMPLATE:
```diff
@@ -3,9 +3,9 @@ Congratulations! You're submitting your assignment.
-|  Do you feel like you used consistent formatting throughout your code? 	|   	|
+|  What was the purpose of the `initialize` method in your class?	|   	
+|  Describe an instance variable you used and what you used it for.	|   	
+|  Describe what the difference would be if your `SolarSystem` used an `Array` vs a `Hash`. 	|   	
```
Sorry to nitpick, but can you get rid of this trailing whitespace as well?

I believe Atom has an option to do this automatically.

CheezItMan committed:
Update feedback.md

CheezItMan committed:
Update PULL_REQUEST_TEMPLATE

CheezItMan committed:
Removed excessive whitespace

Left in 1 space at the end simply because I hate having text squished against the pipe.

CheezItMan committed:
Update PULL_REQUEST_TEMPLATE

CheezItMan commented:
Left 1 space after the "|" simply because I hate text squished against it.
